JavaDirectory/src/main/java/com/bug/free/invention/api/Models/BandD.java

```diff
@@ -0,0 +1,94 @@
+package com.bug.free.invention.api.Models;
```

Comment:
what is the difference between Band and BandD? do we need both, if not remove one